src/search/parser/abstract_syntax_tree.cc

```diff
@@ -147,22 +132,20 @@ FunctionCallNode::FunctionCallNode(const string &name,
 
 static DecoratedASTNodePtr decorate_and_convert(
     const ASTNode &node, const plugins::Type &target_type, DecorateContext &context) {
-    context.push_layer("Decorating node");
```

Comment:
No TraceBlock for this?

Reply:
This function is called at three similar locations to do the same processing. Before each of these location comes a TraceBlock which clarifies the current state. 

Old:
->Checking keyword argument ASDF
->Decorating node
-> ...

There was nothing to fail inbetween the two stack entries. Thus, they would (almost [due to an if]) always follow after another. Thus, there is no added value in this output